#! /bin/sh /usr/share/dpatch/dpatch-run
## mp4ff.h_fix.dpatch by  <matthew@bells23.org.uk>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Imported from Ubuntu

@DPATCH@
diff -urNad faad2.tmp~/common/mp4ff/mp4ff.h faad2.tmp/common/mp4ff/mp4ff.h
--- faad2.tmp~/common/mp4ff/mp4ff.h	2007-10-15 00:36:42.000000000 +0100
+++ faad2.tmp/common/mp4ff/mp4ff.h	2007-10-15 16:52:52.000000000 +0100
@@ -37,7 +37,7 @@
 extern "C" {
 #endif /* __cplusplus */
 
-#include "mp4ff_int_types.h"
+#include <stdint.h> 
 
 /* file callback structure */
 typedef struct
